src/Akeneo/Pim/Enrichment/Bundle/Resources/workspaces/enrichment/src/components/completeness/ChannelsLocalesCompleteness.tsx

```diff
@@ -69,7 +69,7 @@ const CompletenessContainer = styled.div`
   row-gap: 40px;
 `;
 
-const ChannelCompleteness = styled.div`
+const ChannelCompletenessContainer = styled.div`
```

Comment:
As we know we are display completeness can we simplify the name?
```suggestion
const ChannelContainer = styled.div`
```